spec/classes/init_spec.rb

```diff
@@ -482,6 +485,7 @@
     it { should contain_file('sshd_config').with_content(/^PermitRootLogin no$/) }
     it { should contain_file('sshd_config').with_content(/^ChallengeResponseAuthentication no$/) }
     it { should contain_file('sshd_config').with_content(/^PrintMotd no$/) }
+    it { should contain_file('sshd_config').with_content(/^PrintLastLog no$/) }
```

Comment:
Sorry about that...I'm just learning. I will add those two lines. Thanks for the feedback.